apps/wallet-dashboard/lib/constants/routes.constants.ts

```diff
@@ -28,9 +28,9 @@ export const ACTIVITY_ROUTE: ProtectedRoute = {
     path: '/activity',
     icon: Activity,
 };
-export const MIGRATIONS_ROUTE: ProtectedRoute = {
-    title: ProtectedRouteTitle.Migrations,
-    path: '/migrations',
+export const MIGRATION_ROUTE: ProtectedRoute = {
+    title: ProtectedRouteTitle.Migration,
+    path: '/migration',
```

Comment:
shouldnt we also update https://github.com/iotaledger/iota/blob/67b1871254c2403c853148370f9559cc9cb82295/apps/wallet-dashboard/components/MigrationOverview.tsx#L26